requirements/requirements.txt

```diff
@@ -0,0 +1,4 @@
+neon_transformers==0.1.0
+ovos_plugin_manager==0.0.17
+ovos_utils==0.0.22
+neon-utils~=0.17,>=0.17.3
```

Comment:
These should be relaxed to:
```
neon_transformers~=0.1
ovos_plugin_manager~=0.0,>=0.0.17
ovos_utils~=0.0,>=0.0.22
neon_utils>=0.17.3,<2.0.0
```

Then this will be installable after those dependencies are updated